scenarioo-server/src/main/java/org/scenarioo/rest/configuration/DiffViewerStatus.java

```diff
@@ -4,13 +4,4 @@
 
 @XmlRootElement
 public class DiffViewerStatus {
```

Comment:
yes, indeed - well spotted! --> that will also have impact on the client side, because there is some config view where this status is displayed - that is dead code that has to be removed everywhere it is still used (in REST service and as well on the client side).